### PR TITLE
tweakers.net - Fix Dynamic on this site

### PR DESCRIPTION
**Pull request by Gusted**
Weird repeat arrow? It's not even suspossed to view for desktop version...
Before
before
After
after

The top bar was? Drunk I guess
Before
before
After
after

More repeating arrows? and the arrow was placed in a sub section of the element instead of the parent element.... 
Before
before
After
after

Box doesn't matter when on darkmode
Before
before
After
after

Couldn't see the replycounts on banners articles
Before
before
After
after

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2347,6 +2347,118 @@ CSS
...
+CSS
+.ankeiler>a.commentCount {
+    background-image: url(../../../g/icons/commentcounter_small_bg.svg) !important;
+    text-decoration: none !important;
+}
+.thumb.category {
+   border: none !important;
+}
+#categoryBrowser li a{
+     background: none !important;
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config]: Use 4 spaces

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -2347,6 +2347,118 @@ CSS
...
+}
+
+#categoryBrowser li.more {
+    background-image: url("../../../../g/if/categories/arrows.png") !important;
+    background-color:
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config]: Missing value

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -2347,6 +2347,118 @@ CSS
...
+#categoryBrowser.sidebar li.more {
+    background-position: 282px 10px;
+}
+
+a.showMoreItems {
+   background-image: none !important;
+}
+
+.corner::before,.corner::after {
+    border: 6px solid #333 !important;
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config]: Please, check how the fix looks in light mode. Use `${#333}` pattern putting a source color into `${}`, later the changed value will be substituted.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: It looks like this ![https://i.imgur.com/jt8OGPJ.png] ${#333}
And like this ![https://i.imgur.com/V2FMjyx.png] with #333 So I don't think it's handy to use ${}
It looks like this in light mode ![https://i.imgur.com/T8aDoYp.png]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Fixed Dev tools Doesn't Support tab? (It just unfocus the page)

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Removed I don't know why it was there

**Comment by alexanderby** [comment]
Great, thanks.

**Comment by Vistaus** [comment]
Thanks a lot! I had the exact same issues (I'm an avid Tweakers member :) ). I tested your changes, but everything looks the same as before, i.e. still broken. So I think you need to tweak (no pun intended) your code some more.

**Comment by Gusted** [comment]
Could you provide a screenshot? Because for me it's now just working as intended.

Did you make sure you hit Apply?

**Pull request merged**